src/App.css

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100&display=swap');
+* {
+    scroll-behavior: smooth;
+}
+
+:root {
+    --white: #ffffff;
+    --black: #000000;
+    --gray: #808080;
+    --background_white: #ddd;
+    --antiquewhite: #faebd7;
+    --error: #aa1a13;
+    --warning: #b4710c;
+    --info: #653088;
+    --sucess: #0fd884;
+    --default_button: #D5D5D5;
+    --primary_button: #ff0044;
+    --primary_button_hover: #ffff14;
+    --sucess_btn: #1ad61a;
+    --error_btn: #e41111;
+    --secondary_btn: #1665a7;
+    --info_btn: #2581cc;
+    --input_shadow: #34bb51;
+    --rating: #ffe400;
+    --navbar: #7952B3;
+}
+
+.App {
+    min-height: 100vh;
+    background-color: #39445a;
+    color: white;
+    padding-top: 140px;
+    padding-bottom: 70px;
+}
+
+@media(max-width:700px) {
```

Comment:
The heading (series, heading, trending) for each page is getting hidden by the top nav.